daemon/containerd/image_exporter.go

```diff
@@ -342,3 +378,62 @@ func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, platf
 
 	return nil
 }
+
+// verifyImagesProvidePlatform checks if the requested platform is loaded.
+// If the requested platform is not loaded, it returns an error.
+func (i *ImageService) verifyImagesProvidePlatform(ctx context.Context, imgs []containerdimages.Image, platform ocispec.Platform, pm platforms.Matcher) error {
+	var imageNames []string
```

Comment:
Perhaps good to name this `missingImageNames` (I had to read the last part of this function twice to get that this slice _NOT_ being empty is the error condition)